### PR TITLE
Expand introduction section

### PR DESCRIPTION
**Pull request by JKRhb**
This contains an initial attempt at expanding the introduction section. There is still some work to be done, but I think it could already be reviewed. I will try to finish it until the meeting on Monday.

~~I also had to remove the IMAGES file in this PR in order to be able to push the commit. Let me know if the file is too important to be deleted, then I will try to find a different solution for submitting this PR.~~

***
<a href="https://pr-preview.s3.amazonaws.com/JKRhb/wot-security/pull/217.html" title="Last updated on Mar 13, 2023, 12:34 PM UTC (f65b7c8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-security/217/66d919a...JKRhb:f65b7c8.html" title="Last updated on Mar 13, 2023, 12:34 PM UTC (f65b7c8)">Diff</a>

### CODE REVIEW

**Comment by mmccool** [comment]
Unfortunately, I don't think we want to delete the IMAGES directory under note3 since it was a record of a previous publication.  Can you do a new PR with just the changes to index.html?   I don't see why github is bothering you if you don't modify those files...

File: index.html [hunk 1]
```diff
@@ -546,9 +564,28 @@
...
-      the security of the Thing the Thing Description describes.
-    <section>
+      <p>
+        Security and privacy are crucial requirements for Web of Things:
+        Using WoT-based technologies must not put the integrity of a system or the confidentiality of personal information at risk.
+        This document wants to help implementors and stakeholders in fulfilling these important requirements and provides guidelines for secure and privacy-respecting deployments.
+      </p>
+      <p>
+        Definition-wise, this document builds upon the [[WoT-Architecture]] and the ISO definitions for security [[ISO-IEC-27000]] and privacy ([[ISO-IEC-2382]], [[ISO-IEC-29100]]) it is referring to.
+        The informative guidelines defined here accompany the other deliverables published by the WoT Working Group and especially the normative [[WoT-Architecture]],  [[WoT-Thing-Description]], and [[WoT-Discovery]] documents (also see section [[[#related-documents]]]).
```
Comment (mmccool) [inline index.html:574]: Probably need to reference the new versions of the Arch and TD specs?

File: index.html [hunk 2]
```diff
@@ -546,9 +564,28 @@
...
+      </p>
+      <p>
+        <!-- TODO: Revisit document overview -->
```
Comment (mmccool) [inline index.html:577]: ok, but we have to be careful this is not too redundant.

**Review by mmccool — COMMENTED** [review]
I'm ok with merging this, as it's an improvement, but need to think about whether we want to repeat the description of the section structure.

**Comment by JKRhb** [comment]
> Unfortunately, I don't think we want to delete the IMAGES directory under note3 since it was a record of a previous publication. Can you do a new PR with just the changes to index.html? I don't see why github is bothering you if you don't modify those files...

Thank you for the feedback! I assume it was a problem specific to the machine I used at the time, on a different system I was now able to push the changes without a change to the IMAGES directory (i.e., I was able to drop the commit deleting it).

File: index.html (hunk 2 above)
Comment (JKRhb) [reply index.html:577]: You are right, we could also remove it again. Do you have any suggestions on what should be added to the introduction instead? Would it make sense to take over some parts of the abstract and in turn make the abstract a bit more concise?

**Comment by mmccool** [comment]
Some improvements needed:
- Put last paragraph in a section, titled "Outline", use a bulleted list to make it easier to read, avoid making descriptions self-referential ("Lifecycle section explains the lifecycle" -> "Lifecycle section explains the process by which things are incorporated into a WoT system...")
- Don't think we so much "build on" the ISO definitions as reference or use them.

Smaller improvements e.g. to wording should be done using github suggestions

**Pull request merged**